Nightly integration failure: GitHub Copilot CLI

Nightly integration tests failed for `copilot`.

| Field | Value |
|---|---|
| Adapter | `copilot` (agent) |
| Tested version | GitHub Copilot CLI 1.0.57. |
| Install source | npm: @github/copilot (unpinned, installs latest) |
| Run | https://github.com/sortie-ai/sortie/actions/runs/26803873050 |
| Commit | `9327e70506241fc615b32ef40309ef0b867115c5` |
| Date | 2026-06-02 06:59 UTC |

### Failing tests

- `TestIntegration_ResumeSession`
- `TestIntegration_ResumeSessionID`
- `TestIntegration_RunTurn`

### Log excerpt

```
=== RUN   TestIntegration_StartSession
--- PASS: TestIntegration_StartSession (1.07s)
=== RUN   TestIntegration_StopSession
--- PASS: TestIntegration_StopSession (1.07s)
=== RUN   TestIntegration_StartSession_InvalidCommand
--- PASS: TestIntegration_StartSession_InvalidCommand (0.00s)
=== RUN   TestIntegration_RunTurn
2026/06/02 06:59:37 WARN agent stderr component=copilot-adapter line="Error: Model \"gpt-4.1\" from --model flag is not available."
    integration_test.go:216: RunTurn: agent: port_exit: exit code 1
--- FAIL: TestIntegration_RunTurn (3.87s)
=== RUN   TestIntegration_RunTurn_ContextCancellation
--- PASS: TestIntegration_RunTurn_ContextCancellation (3.62s)
=== RUN   TestIntegration_ResumeSession
2026/06/02 06:59:44 WARN agent stderr component=copilot-adapter line="Error: Model \"gpt-4.1\" from --model flag is not available."
    integration_test.go:358: RunTurn (turn 1): agent: port_exit: exit code 1
--- FAIL: TestIntegration_ResumeSession (3.34s)
=== RUN   TestIntegration_ResumeSessionID
2026/06/02 06:59:47 WARN agent stderr component=copilot-adapter line="Error: Model \"gpt-4.1\" from --model flag is not available."
    integration_test.go:421: RunTurn (session 1): agent: port_exit: exit code 1
--- FAIL: TestIntegration_ResumeSessionID (3.40s)
FAIL
FAIL	github.com/sortie-ai/sortie/internal/agent/copilot	16.366s
```

_Filed by the nightly integration monitor. It comments on recurrence and closes automatically when this adapter passes again._

--- a/examples/WORKFLOW.copilot.md
+++ b/examples/WORKFLOW.copilot.md
@@ -44,7 +44,7 @@ agent:
   max_retry_backoff_ms: 300000
 
 copilot-cli:
-  model: gpt-4.1
+  model: gpt-5-mini
   max_autopilot_continues: 50
 
 server:

--- a/internal/agent/copilot/integration_test.go
+++ b/internal/agent/copilot/integration_test.go
@@ -36,9 +36,11 @@ func skipUnlessCopilotIntegration(t *testing.T) {
 
 // integrationConfig builds the adapter config map for integration tests.
 func integrationConfig() map[string]any {
+	// For supported models for Copilot CLI, see
+	// https://docs.github.com/en/copilot/reference/ai-models/supported-models#supported-ai-models-in-auto-model-selection
 	model := os.Getenv("SORTIE_COPILOT_MODEL")
 	if model == "" {
-		model = "gpt-4.1"
+		model = "gpt-5-mini"
 	}
 	return map[string]any{
 		"max_autopilot_continues": float64(5),

